lambda/lambda_handler.py

```diff
@@ -0,0 +1,50 @@
+import importlib
+import sys
+import logging
+
+from scanners import utils
+
+# Central handler for all Lambda events.
+def handler(event, context):
```

Comment:
All Lambda function executions begin here, and are then dispatched to the right code file from here. It's up to the Lambda task builder to make sure that this and the scanner file (and `scanner/utils.py`, and the overall set of deps) are present in the task's environment.